apps/test/solutions/applab/ec_design.js

```diff
@@ -61,6 +61,59 @@ var isHidden = function (selector) {
   return rootEl.is(':visible') && element.hasClass('design-mode-hidden');
 };
 
+function resizeElement(element, xAmount, yAmount) {
+  var resizer = $(element).parent().find('.ui-icon-gripsmall-diagonal-se');
+
+  var start = {
```

Comment:
it would be helpful to explain what this is that we're clicking on, e.g. "bottom right corner of the resize icon"